install.md

```diff
@@ -212,6 +212,65 @@ Building in a container is simpler, but more restrictive:
 $ make binary
 ```
 
+### Build and Install shell completion scripts
+
+Skopeo can create shell completion scripts for bash, zsh, fish and powershell.
+
+This places the scripts under the `completions` directory:
+```bash
+$ make completions
+```
+
+If you want to install the files to the system directories (on linux) use:
+```bash
+$ sudo make install-completions
+```
+
+By default the completion scripts will show extra descriptions as help:
+```
+$ skopeo l[TAB]
+list-tags  (List tags in the transport/repository specified by the SOURCE-IMAGE)  logout     (Logout of a container registry)
+login      (Login to a container registry)
+```
+
+To turn this of you can generate the scripts with the `skopeo completion --no-desc` option,
+see below for the individual shell instructions.
+
```

Comment:
Do users care strongly enough about this to make everyone _think_ about making a decision? To an extent, the purpose of `install.md` is make installing Skopeo easy.

Reply:
I cannot answer that. I personally do not know anybody using it but this doesn't say anything.
When the description support was first introduced in cobra I had to use this option because of many bugs with incorrect rendering of the descriptions. I am not aware of any problems with the current cobra completion scripts for all shells so I no longer use it.
But is is a personal choice and upstream provides this option so I see no reason to not expose this option.

Reply:
Given that the shortest installation method is just `make install`, I guess we’d want a Makefile variable to control that flag. (And should that be just `COMPLETION_GENERATION_FLAGS=arbitrary_string`, or actually `COMPLETIONS_NO_DESCRIPTIONS=true`? The latter is more work to implement and document, OTOH invariant to using the upstream or c/common completion generation command.)

Reply:
Not sure if it is worth the added complexity. I would wait until someone actually requests this.

Reply:
Works for me.